examples/mockups/pricing.html

```diff
@@ -0,0 +1,786 @@
+              <div class="ui label ribbon yellow big">Gold</div>
+
+              <div class="ui header huge"><small><i>from</i></small> $5</div>
+              <div class="meta">
+                <span class="category">per month</span>
+              </div>
+
+              <button class="ui button fluid large yellow">Subscribe</button>
+
+              <div class="description">
+                
+                <div class="ui list relaxed divided">
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Automatic build</div>
+                      <div class="description">on every commit</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Multi-version handling</div>
+                      <div class="description">from branches or tags</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Downloadable formats</div>
+                      <div class="description">PDF, HTML</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Enhanced full-text search</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Ad-free user</div>
+                      <div class="description">stop seeing ads on any Read the Docs hosted documentation when logged in your account</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Ad-free documentation</div>
+                      <div class="description">stop showing ads to all your users on your Read the Docs hosted documentation</div>
+                    </div>
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+
+            <div class="extra content">
+              <span class="left floated share">
+                <i class="icon share alternate"></i>
+                Share
+              </span>
+              <a class="right floated contact">
+                Learn more
+              </a>
+            </div>
+
+          </div>
+          <!-- end card -->
+          
+        </div>
+
+        <div class="ui horizontal divider header">
+          <i class="icon building"></i> Read the Docs
+        </div>
+        <h2 class="ui header centered">
+          <span class="ui header large">for Business</span>
+        </h2>
+        
+        <div class="ui three stackable cards">
+
+          <!-- Card -->
+          <div class="ui inverted card">
```

Comment:
I did some tests with inverting the backgrounds instead of inverted the cards, but I thought that inverting just the cards was the best way of transmitting the difference between the plans, while still maintaining a common thread.

Or do you mean adding a background in addition to the inverted cards? Will that not create too much color noise to allow good legibility? I'll do some tests.

Reply:
> Or do you mean adding a background in addition to the inverted cards? Will that not create too much color noise to allow good legibility? I'll do some tests.

Yeah, I was thinking testing a background color behind the inverted cards. Just a thought, I'm not sure I'm enthusiastic about this.

Reply:
Opened issue #13 to test this further.